package.json

```diff
@@ -11,9 +11,10 @@
     "url": "https://github.com/simple-icons/simple-icons/issues"
   },
   "scripts": {
+    "build": "node scripts/build-pdfs.js",
+    "clean": "rm -rf icons",
```

Comment:
I think this should be renamed to `prebuild`. I'd say `clean` is generally reserved for cleaning up generated artefacts that are not part of version control. Moreover, this command should only ever be run before `build` so naming it as such makes sense.

If this is updated you can remove the `npm run clean` command from the workflow file as well.